### PR TITLE
Adds threshold plots to binary classification example

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
Fixes KOL-2793

### What change does this PR introduce and why?
* Bumps up the kolena client version on the classification example 
* Adds threshold plots to binary classification example

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: examples/classification/classification/evaluator_binary.py [hunk 1]
```diff
@@ -123,6 +125,21 @@ def compute_test_case_plots(
                 labels=[positive_label, negative_label],
             ),
         )
+
+        threshold_curves = compute_threshold_curves(
```
Comment (marko-polo-cheno) [inline examples/classification/classification/evaluator_binary.py:130]: the [return type](https://github.com/kolenaIO/kolena/blob/15b2ae3c976074197ea26e40214d2e46cd60d656/kolena/_experimental/classification/utils.py#L263C9-L263C9) of `compute_threshold_curves` is slightly off, but rest looks good!

**Review by marko-polo-cheno — APPROVED** [review]

File: examples/classification/classification/evaluator_binary.py (hunk 1 above)
Comment (y27choi) [reply examples/classification/classification/evaluator_binary.py:130]: What do you mean by off?

**Comment by gordonhart** [comment]
Plot looks good. Two questions:

1. Do you think it would be a better UX to split each curve into its own plot? The single plot can get quite noisy with multiple test cases plotted.
2. Is accuracy intentionally omitted?

+1 on @marko-polo-cheno's comment about the return type as well.

**Comment by y27choi** [comment]
> 1. Do you think it would be a better UX to split each curve into its own plot? The single plot can get quite noisy with multiple test cases plotted.

How about I split them into F1-threshold and PR curve?

> 2. Is accuracy intentionally omitted?

Intentionally left out accuracy as precision, recall and f1 scores are more meaningful metrics.

File: examples/classification/classification/evaluator_binary.py (hunk 1 above)
Comment (marko-polo-cheno) [reply examples/classification/classification/evaluator_binary.py:130]: `Optional[List[CurvePlot]]` -> `Optional[List[Curve]]`

Comment (y27choi) [reply examples/classification/classification/evaluator_binary.py:130]: OH! 🙈

**Comment by y27choi** [comment]
Updated the return type and separated the plot out to 2 plots (F1 vs. T and PR).
[Example](https://trunk.kolena.io/yoohee/results?modelIds=684&testSuiteId=1371&workflow=Image+Classification&evaluators=N4IgjAzA7GIFygGwA4As8wAYCciC%2BeQA)

**Review by gordonhart — APPROVED** [review]

**Pull request merged**